irc/client.py

```diff
@@ -844,8 +848,8 @@ def links(self, remote_server="", server_mask=""):
     def list(self, channels=None, server=""):
         """Send a LIST command."""
         command = "LIST"
-        channels = ",".join(always_iterable(channels))
-        if channels:
+        if channels != None:
```

Comment:
If channels is "abc", always_iterable returns ("abc",), but this code will use ("a", "b", "c").